pkg/skyenv/skyenv.go

```diff
@@ -108,7 +108,8 @@ const (
 	RewardFile string = "reward.txt"
 
 	// NodeInfo is the name of the survey file
-	NodeInfo string = "node-info.json"
+	NodeInfo   string = "node-info.json"
+	SurveyName string = "node-info" // SurveyName ...
```

Comment:
I've dropped the extra declaration SurveyName, but instead of directly hardcoding I've opted for this implementation:

```
	//Derive the surveyName from visorconfig.NodeInfo by dropping the ".json" suffix
	surveyName := strings.TrimSuffix(string(visorconfig.NodeInfo), ".json")
	// This survey endpoint generates the survey as a response
	authRoute.GET("/"+surveyName, func(c *gin.Context) {

```